### DISCUSSION TITLE
Stack Effect

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13530] [Q&A]
I have some confusion regarding the neutral plane and its relation to the stack effect and the pressure reference point in my model. I’m simulating a fire in an apartment at ground level, with the environmental conditions unchanged. There’s a natural opening at the head of the stair (an open boundary condition), which acts as an inlet for air, while smoke rises through a natural shaft with an opening at the top (an open boundary condition). The velocity vectors seem accurate, with airflow entering through the stair opening and exiting from the top of the shaft.

However, I'm puzzled by the pressure slice through the shaft. The pressure appears highest at the top of the shaft, but shouldn’t there be a neutral plane where the pressure inside the shaft becomes higher than at the top? Also, does FDS account for a reference point, so that even though I’ve set the top as an open boundary condition, the pressure should still be lower than atmospheric pressure due to the height of the building? If flow moves from high pressure to low pressure, how is the pressure decreasing upwards, yet the flow direction is upwards?

![{AB83E9EE-8CA8-4730-A375-A47705E6A25C}](https://github.com/user-attachments/assets/dc328491-b0eb-48c4-9b9d-68fd0d34afa7)

**Comment by mcgratta** [comment]
I suggest you perform a simulation with no fire and no ventilation. Look at `PRESSURE` and `ABSOLUTE PRESSURE`. Sometimes the pressure plot can be confusing because it is a perturbation to the background pressure. `ABSOLUTE PRESSURE` should give you the actual pressure, including the atmospheric stratification.

**Reply by ShehabGamrah** [reply]
Thanks! I’ve outputted absolute pressure for a similar scenario where there’s an exhaust surface at the lowest level, and 100 meters above, an opening in the stairwell that creates a downward flow of air to supply the exhaust. When looking at the regular pressure slice, the pressure is higher at the top and lower at the exhaust, so the air flows from high to low pressure, with a delta P of 3 Pascals, which makes sense. However, the absolute pressure plot shows the opposite, with lower pressure at the top and higher at the bottom, which goes against the flow direction (air flowing from high to low pressure), and the difference is over 1000 Pascals. I'm unsure which plot to rely on.

![{75DE2BA9-C227-4A90-B1E8-8EFADDEB0D85}](https://github.com/user-attachments/assets/065f988d-5935-47b8-b528-6f0f144ebcc2)
![{BB04E36A-4F0B-4B7D-9CCF-888BF42EC802}](https://github.com/user-attachments/assets/141f322e-06bd-4f18-8af0-efe6b18e7384)
